tls/extensions/s2n_client_psk.c

```diff
@@ -294,8 +295,8 @@ static S2N_RESULT s2n_client_psk_recv_binder_list(struct s2n_connection *conn, s
         RESULT_GUARD_POSIX(s2n_blob_init(&wire_binder, wire_binder_data, wire_binder_size));
 
         if (wire_index == conn->psk_params.chosen_psk_wire_index) {
-            RESULT_GUARD_POSIX(s2n_psk_verify_binder(conn, conn->psk_params.chosen_psk,
-                    partial_client_hello, &wire_binder));
+            RESULT_GUARD_POSIX(
+                    s2n_psk_verify_binder(conn, conn->psk_params.chosen_psk, partial_client_hello, &wire_binder));
```

Comment:
I wonder if we should just give up and turn line wrap off. The formatter seems surprisingly bad at line wrapping, and that's probably the formatting nit I have to comment on the least.

Reply:
Col limit 0 to the rescue.

Reply:
It's a shame we have to turn that off :( But people are generally pretty good at sane linewrapping, and if we figure out a better combo of settings we can always apply that later.